packages/frontend/src/app/shared/navbar/language-selector/language-selector.component.scss

```diff
@@ -0,0 +1,42 @@
+@import "~@angular/material/theming";
+
+$flag-filter-shadow: drop-shadow(0 0 2px rgba(0, 0, 0, 0.5));
+$flag-font-size: 25px;
+
+.language-selector {
+  &__button {
+    font-size: $flag-font-size;
+    filter: $flag-filter-shadow;
+  }
+
+  ::ng-deep &__menu {
```

Comment:
I think we could transfer it to global styles.scss as   ".language-selector__menu.mat-menu-panel"  or even to seperated file in global styles  like _navbar.scss or something and attach it to styles? What do you think? Or at least combine it with :host.

Reply:
I will use :host